streamer/controller_node.py

```diff
@@ -259,7 +303,8 @@ def _append_nodes_for_inputs_list(self, inputs: List[Input], output_location: st
 
     self._nodes.append(PackagerNode(self._pipeline_config,
                                     output_location,
-                                    outputs))
+                                    outputs,
+                                    self.hermetic_pacakger))
```

Comment:
typo: packager

Though, no joke, as I tried to type "typo" in that comment, I first accidentally typed "topy".  😄